examples/ucnf-kiknos/README.md

```diff
@@ -0,0 +1,191 @@
+# Kiknos base VPN NSE
+
+This replaces the VPP agent in the universal-cnf with the Kiknos VPP aio-agent
+
+# Prerequisites
+You first need to clone the [Network Service Mesh repo](https://github.com/networkservicemesh/networkservicemesh)
+Please follow the instructions on where the NSM project should be in the [Examples README.md](../../README.md)
+
+- helm v2.16.3
+- kubectl v1.18.2
+
+##### Kind deployment
+- kind v0.7.0 
+
+##### AWS deployment
+- aws-cli v2.0.11
+- eksctl v0.18.0
+- python >= 2.7
+
+ 
+ # Scenarios
+ ## 1. Direct connection between workloads and NSE
+ In this case, the NSE connects directly to the workloads (pods) deployed on the cluster. 
+ With this configuration, all the workloads can be accessed from outside the cluster using the IP address provided by 
+ the NSM through a secure IP Sec tunnel.  
+ 
+ ## 2. Using an Ingress Gateway as NSC
+ Expose the workloads through an [Istio](https://istio.io/) Ingress Gateway. In this case the Ingress Gateway will act 
+ as a network service client. With this configuration, it is easy to expose workloads through a common entry point. This 
+ allows for the case where you want all workloads to be exposed through a single IP address and differentiate between them.
+ This case provides better configurability of the k8s cluster, but adds a layer of complexity with the introduction of Istio.
+
+ 
+# Testing
+
+> :warning: **For the current version build image are available only with `ORG=vladcodaniel` env **
```

Comment:
We will want to build and tag a set of images that we can give to Lunarway and reproduce if needed.

Reply:
This is a older description, I removed it and changed the default `ORG` value to `tiswanso`. Also I documented how user can build it's own images using `make k8s-ucnf-kiknos-save` command